commands/permissions_test.go

```diff
@@ -187,3 +187,70 @@ func (s *MmctlUnitTestSuite) TestRemovePermissionsCmd() {
 		s.Require().EqualError(err, "Role: role_not_found, ")
 	})
 }
+
+func (s *MmctlUnitTestSuite) TestResetPermissionsCmd() {
```

Comment:
Can you please add a couple of end to end tests to the PR?

Reply:
> Could you please add a couple of end to end tests to the PR?

What's the intent you're thinking? To ensure that `PatchRole` does what it should?

Reply:
Mainly the communication between the tool and the server, yes. All the permutation tests like args, errors and such should be tested in the unit test layer, and for the end to end we're trying to add a happy path case to ensure that we're not missing anything with the mocks and a permissions error case if there is any.

On top of those we can add more cases if they are particularly interesting to test for a given command, but if not these two are enough.